how-tos/create-and-run-migrations.md

```diff
@@ -0,0 +1,43 @@
+# How to create and run migrations
```

Comment:
I had written [this doc](https://github.com/reactioncommerce/docs/blob/trunk/guides/API-plugin-package-migration.md) a few months ago with more detailed steps. It could add to this doc.

Reply:
I stole some stuff from that doc and fixed a couple of issues with it. Take another read and see if it makes sense/is accurate. Generally I think this whole migration process needs a lot of work but we can go back to that.